pkg/util/proto/openapi_test.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 var fakeSchema = testing.Fake{Path: filepath.Join("testing", "swagger.json")}
+var fakeSchemaNext = testing.Fake{Path: filepath.Join("testing", "swagger_next.json")}
```

Comment:
I'm also writing a change to that code, and I don't want to create a new swagger file for every new piece of feature. Can we just have a new type that uses your feature?